wordpress-mobile/WordPress-FluxC-Android
# Woo: Countries and states

This PR adds the endpoint that provides a list of countries and states.

**To test:**
1. Start the Example app
2. Tap on the Woo button
3. Tap on the Countries button
4. Wait for the data to load
5. Notice a list of countries is printed

**Unit tests:**
Run the tests in **WCDataStoreTest**.
